CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Analysis/AnalysisScreenLoadingIndicatorAdapter.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  AnalysisScreenLoadingIndicatorAdapter.swift
+//  
+//
+//  Created by David Vizaknai on 12.09.2022.
+//
+
+import Foundation
+/**
+*   Adapter for injecting a custom loading indicator for the analysis viewcontroller.
+*/
+public protocol AnalysisScreenLoadingIndicatorAdapter: InjectedViewAdapter {
+    /**
+     *  Called when the screen is loaded, can be also called from the AnalysisViewController's showAnimation() method.
+     */
+    func startAnimation()
+    /**
+     *  Can be called from the AnalysisViewController's hideAnimation() method.
```

Comment:
Similarly to the above comment my change proposal would be:
```suggestion
     *  Called when the screen has disappeared. You should stop the loading indicator animation in this method.
```